src/main/java/menion/android/whereyougo/preferences/PreferenceValues.java

```diff
@@ -13,6 +13,11 @@
  * see <http://www.gnu.org/licenses/>.
  * 
  * Copyright (C) 2012 Menion <whereyougo@asamm.cz>
+ *
+ * Changes:
+ * Date        Who        Detail
+ * 19.01.2018  Kurly1     Replaced enableWakeLock function tu support partial WakeLock
+ *                        to allow the application to run if screen is off.
```

Comment:
We use the git history to find out this infos. 
Please remove.

Reply:
I removed the change log from the files.